README.md

```diff
@@ -60,21 +60,26 @@ longer track you using cookies or JavaScript. Your subscriptions and history are
 * Show/hide functionality or elements within the app using the distraction free settings
 * View channel community posts
 
-### Browser Extension
-FreeTube is supported by the [Privacy Redirect](https://github.com/SimonBrazell/privacy-redirect) and [LibRedirect](https://github.com/libredirect/libredirect) extensions, which will allow you to open YouTube links into FreeTube. 
+### Browser Extensions
+The following extensions open YouTube links directly in FreeTube:
 
-> [!IMPORTANT]  
-> You must enable the option within the advanced settings of the extension for it to work.
+- [LibRedirect](https://libredirect.github.io/)
+- [RedirectTube](https://github.com/MStankiewiczOfficial/RedirectTube)
+
+LibRedirect automatically redirect YouTube links to FreeTube. To ensure proper functionality, select FreeTube as Frontend in the Services settings of the extension.
```

Comment:
Hmm maybe this should be highlighted as important

"To ensure proper functionality, select FreeTube as Frontend in the Services settings of the extension."

Reply:
Good idea 👍